src/components/UserInfo/UserInfo.jsx

```diff
@@ -2,0 +1,11 @@
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <p>
+    {' Posted by  '}
+
+    <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
Ensure that the `user.email` value is sanitized or validated to prevent potential security issues, such as injection attacks, when used in the `href` attribute.